reana_client/cli/files.py

```diff
@@ -155,8 +156,20 @@ def get_files(ctx, workflow, _filter,
     help='Path to the directory where files will be downloaded.')
 @add_access_token_options
 @click.pass_context
-def download_files(ctx, workflow, filenames, output_directory, access_token):
-    """Download workflow workspace file(s)."""
+def download_files(ctx, workflow, filenames,
+                   output_directory, access_token):  # noqa: D301
+    """Download workflow output files.
```

Comment:
I think it is time to just say `Download workflow [workspace] files.` since any file can be download and it might make sense to download intermediate or even input files.

I would also rephrase to make the default behaviour more obvious to the reader:
```console
"""Download workflow workspace files.

This command allows downloading any file from a workflow workspace. By default if no
arguments are provided (1st example below), files which are declared in workflow
specification as output will be downloaded. Note that downloading directories is not
yet supported.

....
"""
```